sig-docs/english-doc-list/KubeSphere 3.0 Documentation List.md

```diff
@@ -0,0 +1,78 @@
+# KubeSphere 3.0 Documentation List
+
+Below are lists of guides that need to be written as part of KubeSphere documentation 3.0. Please add your name under **Writer** to claim the task. When you submit your pr, put the PR link under **PR Link**.
+
+## DevOps User Guide & Administration
+
+### Overview
+
+| Guide                                                        | Writer | PR Link |
+| ------------------------------------------------------------ | ------ | ------- |
+| Introduction                                                 |        |         |
+| Create a Pipeline - using Graphical Editing Panel            |        |         |
+| Create a Pipeline - using Jenkinsfile                        |        |         |
+| Pipeline Management (including webhook, Timing trigger, etc.） |        |         |
+| Credential Management                                        |        |         |
+| Role and Member Management                                   |        |         |
+| Artifactory Management                                       |        |         |
+| Add new types of pod for pipeline                            |        |         |
+
+### How to Integrate
+
+| Guide                                    | Writer | PR Link |
+| ---------------------------------------- | ------ | ------- |
+| Integrating SonarQube in Pipeline        |        |         |
+| Integrating Harbor in Pipeline           |        |         |
+| Integrating Nexus in Pipeline            |        |         |
+| Integrating Jrog Artifactory in Pipeline |        |         |
+
+### Example
+
+| Guide                                                  | Writer | PR Link |
+| ------------------------------------------------------ | ------ | ------- |
+| How to compile and deploy a maven project              |        |         |
+| How to compile and deploy a spring cloud project       |        |         |
+| How to compile and deploy a nodejs project             |        |         |
+| How to compile and deploy a go project                 |        |         |
+| How to compile and deploy a python project             |        |         |
+| How to run automated testing                           |        |         |
+| How to deploy applications cross multiple clusters     |        |         |
+| How to deploy a Micro service application based on K8s |        |         |
+
+## App Store
+
```

Comment:
As for the built-in apps, in addition to basic deployment on KubeSphere, users should also be instructed on how to expose the app's service to other services or expose its dashboard. Some apps need to be provided with simple demos.

Reply:
Writers can add this part in the same guide directly. There is no point writing two separate guides related one app. Otherwise, readers need to switch between guides just for the same app.